schema/mysql/schema.sql

```diff
@@ -1013,7 +1015,9 @@ CREATE TABLE downtime_history (
   trigger_time bigint unsigned NOT NULL,
   cancel_time bigint unsigned DEFAULT NULL,
 
-  PRIMARY KEY (downtime_id)
+  PRIMARY KEY (downtime_id),
+
+  INDEX idx_downtime_history_start_time_end_time (start_time, end_time) COMMENT 'Filter for history retention'
```

Comment:
As already stated in a previous comment, isn't the check for `start_time` redundant and `end_time` could be used as the only value for filtering and thus would be better suited as the first (or only) value in the index? Same for the folliwing indices.

Reply:
Fixed.